### PR TITLE
Make  template format

### PR DESCRIPTION
**Pull request by victoriabernard92**
Sven's fork has changes that make this is appropriate template format
e.g.
* adds package.json
* updates readme to use wrangler
 
TODO: 

- [ ] add wrangler.toml
- [ ] update documentation to use cloudflare github link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
+#### Wrangler
 
-## Usage
+To generate using [wrangler](https://github.com/cloudflare/wrangler)
 
-Define your origin by changing the URL here: https://github.com/xtuc/binast-cf-worker/blob/c7b354bcbc7764faf0df5ee98463df08e453c776/index.js#L2. By default it will serve https://github.com/xtuc/binjs-demo.
-
-The Worker will serve your website and BinAST if supported.
+```
+wrangler generate binast-cf-worker https://github.com/xtuc/binast-cf-worker-template
```
Comment (xtuc) [inline README.md]: Shouldn't this be Cloudflare?

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,4 @@
 node_modules
-dist
```
Comment (EverlastingBugstopper) [inline .gitignore]: why remove dist? gitignore should remain unchanged i think

File: package.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  "version": "1.0.0",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "jpeg-js": "^0.3.5",
+    "pngjs": "^3.4.0",
+    "string-to-stream": "^1.1.1"
+  },
+  "description": "A template for kick starting a Cloudflare Workers project",
```
Comment (victoriabernard92) [inline package.json]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (victoriabernard92) [inline README.md]:
```suggestion
wrangler generate binast-cf-worker https://github.com/cloudflare/binast-cf-worker
```

**Pull request merged**